src/base/types.h

```diff
@@ -35,7 +35,7 @@ const qlonglong MAX_ETA = 8640000;
 
 enum class ShutdownAction
 {
-    None,
+    Exit,
```

Comment:
Now you showed me that the both cases can confuse. I meant the old identifier. I think you are right and this is the enum name which gives the ambiguity. `ShutdownDialogAction` is better.